tests/unit/concurrently/test_TaskPackageDropbox_receive.py

```diff
@@ -87,10 +87,7 @@ def test_receive(obj, pkgidx_result_pairs):
 
 def test_receive_dispatcher_received_failed_runids(obj, dispatcher):
     obj.receive()
-    assert [
-        mock.call([]), mock.call([]), mock.call([1002]),
-        mock.call([1005]), mock.call([])
-    ] == dispatcher.failed_runids.call_args_list
```

Comment:
why did the expected value of this assertion change? my naive expectation was that the existing tests would all pass without the change.

Reply:
I could pass the resubmission responsibility up to `receive` and `receive_one`, then the `call_args_list` of `dispatcher.failed_runids` will be the same as before

Reply:
I see. Thank you for the answer. It is ok as long as `1002` and `1005` are there.